src/compile/mark/mark.ts

```diff
@@ -79,12 +90,10 @@ function parsePathMark(model: UnitModel) { // TODO: extract this into compilePat
     return [{
       name: model.name('pathgroup'),
       type: 'group',
-      from: extend(
-        // If has facet, `from.data` will be added in the cell group.
-        // Otherwise, add it here.
-        isFaceted ? {} : dataFrom,
-        {transform: transform}
-      ),
+      from: {
+        data: dataFrom(model),
+        transform: transform
```

Comment:
I thought transform is not possible any more. Or did you just leave this in here for later?